state-chain/pallets/cf-swapping/src/tests.rs

```diff
@@ -688,6 +688,14 @@ mod ccm {
 	#[track_caller]
 	fn init_ccm_swap_request(input_asset: Asset, output_asset: Asset, input_amount: AssetAmount) {
 		let ccm_deposit_metadata = generate_ccm_deposit();
+		let ccm_deposit_metadata_encoded = cf_chains::CcmDepositMetadataEncoded {
```

Comment:
We could have a method on `CcmDepositMetadata` to convert it into the encoded verson (taking AddressConverter as a generic parameter?). I think we already do this for some of the other structs.